docs/tutorials/manifest-parser.md

```diff
@@ -157,14 +157,14 @@ Media segment times are all in terms of the presentation timeline.  So if the
 content was originally multi-period, the period start time must be accounted for
 in the reference's `timestampOffset` field.
 
-First we ask for the index that corresponds with a start time.  Then on update,
+First, we ask for the index that corresponds with a start time.  Then on the update,
```

Comment:
"on the update" kind of suggests that the manifest will only ever update once. I'd leave that phrasing as-is.

Reply:
Understood!